packages/runtime/src/useCases/common/RuntimeErrors.ts

```diff
@@ -189,6 +190,13 @@ class Attributes {
             `Attribute '${attributeId.toString()}' is not a third party owned relationship attribute.`
         );
     }
+
+    public hasSuccessor(predecessor: LocalAttribute): ApplicationError {
```

Comment:
Why didn't we already have to use such an error for the `SucceedRepositoryAttributeUseCase`? How is the case of trying to succeed a predecessor RepositoryAttribute (which already has a successor) handled and what error is thrown?

Reply:
In `SucceedRepositoryAttributeUseCase` the validation from the AttributesController is called and in case of an error the consumption error is returned.